vita3k/host/src/load_self.cpp

```diff
@@ -42,6 +42,7 @@
 #include <iomanip>
 #include <iostream>
 #include <fstream>
+#include <sys/mman.h>
```

Comment:
?

Reply:
Don't include unnecessary stuff

Reply:
yeah, sorry, I did not notice I include this header, just forgot to remove it after debug. It does not matter to remove it.